docs/source/installation.rst

```diff
@@ -46,6 +46,20 @@ With Git/GitHub
 
 		pip install .
 
+CRDS Environment Variables
+--------------------------
+
+Eureka! installs the JWST Calibration Pipeline as part of its requirements, and this also requires users to set the proper environment variables so that it can download the proper reference files needed to run the pipeline. 
+For users not on the internal STScI network, two environment variables need to be set to enable this functionality:
+
+	.. code-block:: bash
+
+		export CRDS_PATH=/PATH/TO/FOLDER/crds_cache
```

Comment:
It's likely worth specifying that these need to be in the user's .bashrc or similar file (e.g. .bash_profile) otherwise they're only active for that particular terminal instance

Reply:
Ah, you're totally right. I'll add that.